src/chaplin/mediator.coffee

```diff
@@ -47,13 +46,11 @@ mediator.setHandler = (name, method, instance) ->
   handlers[name] = {instance, method}
 
 # Retrieves a handler function and executes it.
-mediator.execute = (nameOrObj, args...) ->
-  silent = false
-  if typeof nameOrObj is 'object'
-    silent = nameOrObj.silent
-    name = nameOrObj.name
+mediator.execute = (options, args...) ->
+  if options is Object(options)
```

Comment:
`x === Object(x)` is used to test method parameters if they are not primitives and the change does not introduce any backwards-incompatible changes. I am trying to avoid throws on passed `null`.

Functions are just objects that can be called. If we are going to treat them as objects (pass multiple parameters, use as hashes) this check is ok. If we are going to call some value, then it is `typeof is 'function'`